tests/coverage/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+zephyr_sources_ifdef(CONFIG_COVERAGE_HW_TEST coverage.c)
```

Comment:
Just noticed that in commit called "tests: coverage: Add Gcov support needed by hardware platforms.", the github link is incorrect, it should be https://github.com/eDeviser/lwgl